src/rax.c

```diff
@@ -507,11 +508,17 @@ int raxGenericInsert(rax *rax, unsigned char *s, size_t len, void *data, void **
      * our key. We have just to reallocate the node and make space for the
      * data pointer. */
     if (i == len && (!h->iscompr || j == 0 /* not in the middle if j is 0 */)) {
+        size_t oldalloc = 0;
+
         debugf("### Insert: node representing key exists\n");
         /* Make space for the value pointer if needed. */
         if (!h->iskey || (h->isnull && overwrite)) {
+            oldalloc = rax_alloc_size(h);
```

Comment:
It's only used within the `if` so better declare it in the `if` scope.

```suggestion
        debugf("### Insert: node representing key exists\n");
        /* Make space for the value pointer if needed. */
        if (!h->iskey || (h->isnull && overwrite)) {
            size_t oldalloc = rax_alloc_size(h);
```

Reply:
Thank you I missed that. Also realizing for next time I can just click the "Sign off and commit suggestion" button :)

Reply:
Yeah "Sign off and commit suggestion" seems to be a new GitHub feature. Very useful for us.